cms/templates/howitworks.html

```diff
@@ -146,7 +163,7 @@ <h2 class="sr">${_("Sign Up for {studio_name} Today!").format(studio_name=settin
 
     <ul class="list-actions">
       <li class="action-item">
-        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up &amp; Start Making an {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
+        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up & Start Making {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
```

Comment:
Maybe try replacing "an" with "your", as in "Sign Up & Start Making your {platform_name} Course".
Otherwise, it will read something like "Sign Up & Start Making Studio Course".